lib/db/schema/patch-034-035.sql

```diff
@@ -0,0 +1,87 @@
+CREATE TABLE IF NOT EXISTS unblockCodes (
+  uid BINARY(16) NOT NULL,
+  unblockCode VARCHAR(32) NOT NULL,
+  createdAt BIGINT SIGNED NOT NULL,
+  UNIQUE INDEX unblockCodes_uid_code(uid, unblockCode)
+) ENGINE=InnoDB;
+
+CREATE PROCEDURE `createUnblockCode_0` (
+    IN inUid BINARY(16),
+    IN inCode VARCHAR(32),
+    IN inCreatedAt BIGINT SIGNED
+)
+BEGIN
+    INSERT INTO unblockCodes(
+        uid,
+        unblockCode,
+        createdAt
+    )
+    VALUES(
+        inUid,
+        inCode,
+        inCreatedAt
+    );
+END;
+
+CREATE PROCEDURE `consumeUnblockCode_0` (
+    inUid BINARY(16),
+    inCode VARCHAR(32)
+)
+BEGIN
+    CREATE TEMPORARY TABLE IF NOT EXISTS unblockTimestamps AS
+    (
+        SELECT createdAt FROM unblockCodes
+        WHERE
+            uid = inUid
+        AND
+            unblockCode = inCode
+    );
```

Comment:
Rather than a temporary table, could we just select this into a local variable and then return it?  I don't recall the exact syntax but it would be something like:

```
declare v_createdAt int default 0;
SELECT createdAt INTO @v_createdAt FROM unblockCodes WHERE ...;
DELETE FROM unblockCodes WHERE ...;
SELECT v_createdAt AS createdAt;
```